src/backupNg/backupNg.spec.js

```diff
@@ -111,5 +111,36 @@ describe("backupNg", () => {
         xo.call("backupNg.runJob", { id })
       ).rejects.toMatchSnapshot();
     });
+
+    it("fails trying to run a backup job with invalid vms pattern", async () => {
```

Comment:
`with a no matching VMs`. Please also update the  `PR` message.

Reply:
This test is about an invalid vms pattern whereas the other one (PR #91) , tests a vm nonexistent (and so : `no matching vm`). Is that what you're talking about?

Reply:
So it have two errors handler by the implementation. 
One using a simple pattern like { id: 'vmId'} in this case a warning is emitted if the VM is missed and another using a more specific pattern like adding VMs properties in this case an error is emitted.

Reply:
Yes.

Reply:
Please update the PR.